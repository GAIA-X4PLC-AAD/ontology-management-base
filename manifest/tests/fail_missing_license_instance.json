{
  "@context": {
    "gx": "https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#",
    "sh": "http://www.w3.org/ns/shacl#",
    "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
    "manifest": "https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/manifest/",
    "xsd": "http://www.w3.org/2001/XMLSchema#",
    "skos": "http://www.w3.org/2004/02/skos/core#"
  },
  "@id": "did:web:registry.gaia-x.eu:Manifest:fail_missing_license_instance",
  "@type": "manifest:Manifest",
  "manifest:manifest": {
    "@type": "manifest:Link",
    "manifest:accessRole": "publicUser",
    "manifest:category": "manifest",
    "manifest:FileMetadata": {
      "@type": "manifest:FileMetadata",
      "manifest:uri": {
        "@value": "./manifest_instance.json",
        "@type": "xsd:anyURI"
      },
      "manifest:mimeType": {
        "@value": "application/json",
        "@type": "xsd:string"
      }
    }
  },
  "manifest:content": [
    {
      "@type": "manifest:Link",
      "manifest:accessRole": "owner",
      "manifest:category": "coreData",
      "manifest:FileMetadata": {
        "@type": "manifest:FileMetadata",
        "manifest:uri": {
          "@value": "ipfs://bafybeiavkhiuq44adcxpuve5eic3tu4ilivuuykzrsyudoivwh3zc2piqy",
          "@type": "xsd:anyURI"
        },
        "manifest:cid": {
          "@value": "bafybeiavkhiuq44adcxpuve5eic3tu4ilivuuykzrsyudoivwh3zc2piqy",
          "@type": "xsd:string"
        },
        "manifest:mimeType": {
          "@value": "application/x-xodr",
          "@type": "xsd:string"
        },
        "manifest:fileSize": 636673,
        "manifest:filename": {
          "@value": "TestfeldNiedersachsen_ALKS_ODR_sample.xodr",
          "@type": "xsd:string"
        }
      }
    },
    {
      "@type": "manifest:Link",
      "manifest:accessRole": "publicUser",
      "manifest:category": "documentation",
      "manifest:FileMetadata": {
        "@type": "manifest:FileMetadata",
        "manifest:uri": {
          "@value": "ipfs://bafkreidjcjbxlyeft6wasxkuerwxvo3acmranffkmv5iy2tiewl2k5z6fe",
          "@type": "xsd:anyURI"
        },
        "manifest:cid": {
          "@value": "bafkreidjcjbxlyeft6wasxkuerwxvo3acmranffkmv5iy2tiewl2k5z6fe",
          "@type": "xsd:string"
        },
        "manifest:mimeType": {
          "@value": "text/markdown",
          "@type": "xsd:string"
        },
        "manifest:fileSize": 5120,
        "manifest:filename": {
          "@value": "README.md",
          "@type": "xsd:string"
        }
      }
    },
    {
      "@type": "manifest:Link",
      "manifest:accessRole": "publicUser",
      "manifest:category": "metadata",
      "manifest:FileMetadata": {
        "@type": "manifest:FileMetadata",
        "manifest:uri": {
          "@value": "ipfs://bafkreifsu25wkyq7oecmtam3mguuoov2zmjp6xi3chuf5plajagdjfoule",
          "@type": "xsd:anyURI"
        },
        "manifest:cid": {
          "@value": "bafkreifsu25wkyq7oecmtam3mguuoov2zmjp6xi3chuf5plajagdjfoule",
          "@type": "xsd:string"
        },
        "manifest:mimeType": {
          "@value": "application/json",
          "@type": "xsd:string"
        },
        "manifest:fileSize": 5678,
        "manifest:filename": {
          "@value": "domainMetadata.json",
          "@type": "xsd:string"
        }
      }
    },
    {
      "@type": "manifest:Link",
      "manifest:accessRole": "registeredUser",
      "manifest:category": "media",
      "manifest:FileMetadata": {
        "@type": "manifest:FileMetadata",
        "manifest:uri": {
          "@value": "./visualization/detailRoadNetwork.geojson",
          "@type": "xsd:anyURI"
        },
        "manifest:cid": {
          "@value": "bafkreiakpb3ajhljq6cgwizcyirsk6uz322szaxd6ynmovcwuu5jkx5isa",
          "@type": "xsd:string"
        },
        "manifest:mimeType": {
          "@value": "application/x-geojson",
          "@type": "xsd:string"
        },
        "manifest:fileSize": 1500,
        "manifest:filename": {
          "@value": "detailRoadNetwork.geojson",
          "@type": "xsd:string"
        }
      }
    },
    {
      "@type": "manifest:Link",
      "manifest:accessRole": "publicUser",
      "manifest:category": "media",
      "manifest:FileMetadata": {
        "@type": "manifest:FileMetadata",
        "manifest:uri": {
          "@value": "./visualization/TestfeldNiedersachsen_ALKS_ODR_sample_01.png",
          "@type": "xsd:anyURI"
        },
        "manifest:cid": {
          "@value": "bafybeiakegwcrpeuylnurqns2pwim67tni6td4jdqosblhwu7kvgvuncqi",
          "@type": "xsd:string"
        },
        "manifest:mimeType": {
          "@value": "image/png",
          "@type": "xsd:string"
        },
        "manifest:dimensions": {
          "@type": "manifest:Dimensions",
          "manifest:unit": "pixels",
          "manifest:width": {
            "@value": "1095",
            "@type": "xsd:decimal"
          },
          "manifest:height": {
            "@value": "850",
            "@type": "xsd:decimal"
          }
        },
        "manifest:fileSize": 2400256,
        "manifest:filename": {
          "@value": "TestfeldNiedersachsen_ALKS_ODR_sample_01.png",
          "@type": "xsd:string"
        }
      }
    }
  ]
}
