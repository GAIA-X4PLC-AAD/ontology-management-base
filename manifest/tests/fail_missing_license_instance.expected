Overall validation explicitly: Conforms=False
Validation Report
Conforms: False
Results (1):
Constraint Violation in MinCountConstraintComponent (http://www.w3.org/ns/shacl#MinCountConstraintComponent):
        Severity: sh:Violation
        Source Shape: [ sh:description Literal("Defines the license(s) valid for all content referenced in the manifest. Does not apply to linked data(sets) with explicit license terms.", lang=en) ; sh:minCount Literal("1", datatype=xsd:integer) ; sh:node manifest:LicenseShape ; sh:order Literal("1", datatype=xsd:integer) ; sh:path manifest:license ]
        Focus Node: <did:web:registry.gaia-x.eu:Manifest:fail_missing_license_instance>
        Result Path: manifest:license
        Message: Less than 1 values on <did:web:registry.gaia-x.eu:Manifest:fail_missing_license_instance>->manifest:license
